front-end/src/reducers/loanReducer.js

```diff
@@ -1,11 +1,14 @@
-import data from '../../helpers/mockData_FE'
-import ACTION_TYPE from '../lib/actionType'
+import { ACTION_TYPE } from '../lib/actions'
 
-function loanReducer(action) {
-  if (action.type === ACTION_TYPE.INIT_LOANLIST) {
-    return data.loans
+const initState = { loanList: [] }
```

Comment:
@JialinWang1 if there's a separate reducer for the loading, then here we just need the list
```suggestion
const initState = []
```